server/src/test/java/org/elasticsearch/indices/memory/breaker/CircuitBreakerServiceIT.java

```diff
@@ -89,7 +89,9 @@ private void reset() {
             HierarchyCircuitBreakerService.FIELDDATA_CIRCUIT_BREAKER_OVERHEAD_SETTING,
             HierarchyCircuitBreakerService.REQUEST_CIRCUIT_BREAKER_LIMIT_SETTING,
             HierarchyCircuitBreakerService.REQUEST_CIRCUIT_BREAKER_OVERHEAD_SETTING,
-            IN_FLIGHT_REQUESTS_CIRCUIT_BREAKER_LIMIT_SETTING,
```

Comment:
Nit: Should this be changed consistently across the whole file so the static import can be removed?

Reply:
Sure, I've updated this to remove the static import